setup-elytron-guides/setup_elytron_linux.adoc

```diff
@@ -0,0 +1,148 @@
+---
+layout: setup-elytron
+title: Setting up WildFly Elytron on Linux & WSL
+permalink: /guides/setup-elytron-linux/
+os: linux
+---
+:imagesdir: ../../assets/images/setup-elytron/
+
+:toc: macro
+:toc-title:
+
+This page will guide you through the steps needed to setup a development environment for WildFly Elytron on Linux, or the https://docs.microsoft.com/en-us/windows/wsl/about[Windows Subsystem for Linux (WSL)]. For WSL developers, all commands should be entered into the WSL terminal _(i.e. not Command Prompt, PowerShell, or Git Bash)_.
+
+toc::[]
+
+== Installing Java
+
+. First, let's check if you already have Java installed.
+    .. Open a terminal, and run `java --version`.
+    .. If the output shows a version number, check to make sure it is Java 11 (e.g., "11.0.15.1"). If so, Java 11 is already installed and you can jump to the *Installing Maven* section. If the output is not Java 11, or if the output is `command not found: java` or a similar message indicating you do not have Java installed, then continue with the following steps.
```

Comment:
Installing Maven should link to https://wildfly-security.github.io/wildfly-elytron/guides/setup-elytron-linux/#installing-maven